increment_and_move.py

```diff
@@ -0,0 +1,36 @@
+import boto3
+import sys
+import time
+
+if __name__ == '__main__':
+    if len(sys.argv) != 3:
+        print("Expected arguments: [first queue name] [second queue name]")
+        print("Got only " + str(len(sys.argv) - 1) + " arguments")
+        exit(-1)
+
+    sqs = None
+    while True:
+        try:
+            sqs = boto3.resource(service_name='sqs', endpoint_url='http://localstack:4576')
+            break
+        except:
+            time.sleep(1)
```

Comment:
Can you please explain why `wait(20)` is more solid solution?

Reply:
@TanVD, I've made all services restart on failure instead of sleeping in code. Is this solution more solid? A lot of exceptions are thrown by localstack layer. It seems like I can do nothing about it except for turning off exceptions for this layer.